tornado_prometheus_exporter/__init__.py

```diff
@@ -0,0 +1,57 @@
+from logging import getLogger
+
+from prometheus_client import Histogram, start_http_server
+from prometheus_client import REGISTRY as DEFAULT_REGISTRY
+from tornado.web import Application as _Application
+
+log = getLogger('tornado_prometheus_exporter')
+
+DEFAULT_PORT = 9099
```

Comment:
Instead of reusing a port, we should [allocate one](https://github.com/prometheus/prometheus/wiki/Default-port-allocations).

Reply:
Honestly, i would rather remove the default than do that. Flask and Django also don't have and I don't see how the port is specific to the programming language/web framework

Reply:
removed the default port